signer/remote_signer.go

```diff
@@ -62,7 +63,7 @@ func NewReconnRemoteSigner(
 
 // OnStart implements cmn.Service.
 func (rs *ReconnRemoteSigner) OnStart() error {
-	go rs.loop()
+	go rs.loop(context.Background())
```

Comment:
Why not have `OnStart()` take a context? Seems like that context would be close to one created in `main()`.

Reply:
`OnStart` is an interface method for the comet service. Removing the dependency on comet is something I'd like to do at some point, or at least removing it for the services and logging.